packages/subgraph/src/dao/utils.ts

```diff
@@ -77,41 +86,65 @@ export function handleAction<
       action.to,
       action.value,
       'Native Token Withdraw',
-      proposalId,
+      transactionActionId,
       index,
       event
     );
     return;
   }
 
-  handleTokenTransfers(action, proposalId, index, event);
+  checkForAndHandleTokenTransfers(action, transactionActionId, index, event);
```

Comment:
My first read of the code block was "what if it isn't a native or token transfer?" But it's not a big deal and I can change it back.

Reply:
Basically, my comment is that `Handle` is a kind of word(at least in my head and what I have seen in other's project) that include both: checking + actual handling.  That's why I suggested to still use `Handle` without `checkFor` because it already implies that it also checks.

Reply:
https://github.com/aragon/osx/pull/562/commits/d1ea0383236c3d9b4b79ff3ecde2050d1b57a2ce#diff-59fc39c600da0bb0faee4224c1c7b51d752b65f6dc62a96c9db16380c96c9bcbL147